CONTRIBUTING.md

```diff
@@ -104,3 +106,31 @@ RUN echo  " node version:    $(node -v) \n" \
   "debian version:  $(cat /etc/debian_version) \n" \
   "user:            $(whoami) \n"
 ```
+
+## Multi-Architecture Support
+
+As of Cypress 10.2.0, `arm64` and `x64` are both supported.
+
+In CI, the images are build and tested in real `arm64` and `x64` architectures. Then, via `binfmt` and `docker buildx`, we build x64 and cross-build arm64 from the same machine, and then publish that image to Docker Hub. The `docker buildx` step runs a second time to push to Amazon ECR:
+
+<!-- diagram generated w/ https://asciiflow.com/ -->
+```text
+┌────────────────────┐     ┌──────────────────┐
+│arm64 build+test job│     │x64 build+test job│
+└──────────┬─────────┘     └────────┬─────────┘
+           │                        │
+┌──────────▼────────────────────────▼─────────┐
+│x64 build+push job                           │
+│  ┌──────────────────────────────────────┐   │
+│  │       build+push to Docker Hub       │   │
+│  └──────────────────┬───────────────────┘   │
+│                     │                       │
+│  ┌──────────────────▼───────────────────┐   │
+│  │         build+push to AWS ECR        │   │
+│  └──────────────────────────────────────┘   │
+└─────────────────────────────────────────────┘
+```
```

Comment:
You could use mermaid to render your diagram.

```mermaid
flowchart TD
arm64[arm 64 build+test job] --> x64push
x64[x64 build+test job] --> x64push
subgraph x64push[x64 build+push job]
dockerHub[build+push to Docker Hub] --> AWS[build+push to AWS ECR]
end
```

Reply:
Nice, didn't know that rendered on GH. Though I think I prefer the plaintext diagrams for things as small as this, with the benefit being that it's readable without a Markdown renderer.